MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -4880,6 +4884,20 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 
 	startAutoGPUCapture(MVK_CONFIG_AUTO_GPU_CAPTURE_SCOPE_DEVICE, _physicalDevice->_mtlDevice);
 
+	if (getMVKConfig().autoGPUCaptureScope == MVK_CONFIG_AUTO_GPU_CAPTURE_SCOPE_ON_DEMAND) {
+		char pipeName[] = "/tmp/MoltenVKCapturePipe-XXXXXXXXXX";
+		mktemp(pipeName);
+		if (mkfifo(pipeName, 0600) < 0) {
```

Comment:
If `errno` contains `EEXIST`, that means there was a collision and you should probably try this again with a new filename.

Reply:
Seems a little defensive to me, but fine.